apps/andi/lib/andi_web/live/data_dictionary/data_dictionary_field_editor.ex

```diff
@@ -17,6 +17,7 @@ defmodule AndiWeb.DataDictionary.FieldEditor do
     id = Atom.to_string(assigns.id)
     form_with_errors = DataDictionaryHelpers.add_errors_to_form(assigns.form)
     field_type = input_value(assigns.form, :type)
+    editing_ingestion? = assigns.dataset_or_ingestion == :ingestion
```

Comment:
Elixir question - Does the ? have special meaning? Is it an optional in case assigns.dataset_or_ingestion is nil?

Reply:
Indicates a boolean: https://hexdocs.pm/elixir/main/naming-conventions.html#trailing-question-mark-foo